gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginRegistryImpl.java

```diff
@@ -377,6 +377,15 @@ public Collection<Plugin> plugins(String type) {
         return plugins.stream().filter(pluginContext -> type.equalsIgnoreCase(pluginContext.type())).collect(Collectors.toSet());
     }
 
+    @Override
+    public Plugin get(String type, String id) {
```

Comment:
I don't how often this happens, or when, maybe if plugin were stored in a map <type, map <id, plugin>> that would be faster.

Reply:
You're right! Given that it can be intensively used, I'll find a way to manage a map instead 👍